extensions/typescript-language-features/src/languageFeatures/refactor.ts

```diff
@@ -187,36 +187,67 @@ class MoveToFileRefactorCommand implements Command {
 		type DestinationItem = vscode.QuickPickItem & { readonly file: string };
 
 		const workspaceFolder = vscode.workspace.getWorkspaceFolder(document.uri);
-		const destinationItems = response.body.files.map((file): DestinationItem => {
-			const uri = this.client.toResource(file);
-			const parentDir = Utils.dirname(uri);
-
-			let description;
-			if (workspaceFolder) {
-				if (uri.scheme === Schemes.file) {
-					description = path.relative(workspaceFolder.uri.fsPath, parentDir.fsPath);
+
+		const quickPick = vscode.window.createQuickPick();
+		const body = response.body;
+		const updateItems = () => {
+			const destinationItems = body.files.map((file): DestinationItem | undefined => {
```

Comment:
This should be optimized to not compute anything if the user has not typed `./` or `../` to start the path